data_io.py

```diff
@@ -0,0 +1,70 @@
+import mimetypes
+import os
+from json import dump
+
+import pandas as pd
+
+
+# Input
+def take_input_data(path: str) -> pd.DataFrame:
```

Comment:
add docstring

Reply:
I don't like random docstrings tbh, we have typehints, comments, readme, adding one docstring would break project convention. I added comments though :)